src/jansson.h

```diff
@@ -319,6 +319,7 @@ json_t *json_deep_copy(const json_t *value) JANSSON_ATTRS(warn_unused_result);
 #define JSON_DECODE_ANY         0x4
 #define JSON_DECODE_INT_AS_REAL 0x8
 #define JSON_ALLOW_NUL          0x10
+#define JSON_DECODE_NUMBER_AS_STRING          0x12
```

Comment:
The value should be `0x20` to not have same bits with other flags.

Reply:
woops... fixed